lib/reindex/s3_bucketd.py

```diff
@@ -350,3 +356,26 @@ def log_report(resource, name, obj_count, total_size):
             update_redis(pipeline, 'accounts', userid, report['obj_count'], report['total_size'])
             log_report('accounts', userid, report['obj_count'], report['total_size'])
         pipeline.execute()
+
+    observed_accounts = set(account_reports.keys())
+    recorded_accounts = set(get_resources_from_redis(redis_client, 'accounts'))
+    recorded_buckets = set(get_resources_from_redis(redis_client, 'buckets'))
+
+    # Stale accounts and buckets are ones that do not appear in the listing, but have recorded values
+    stale_accounts = recorded_accounts.difference(observed_accounts)
+    print('Found %s stale accounts' % len(stale_accounts))
```

Comment:
Small preference thing but not a huge fan of %s syntax and I believe python even recommends not using it now. But depending on your version it might still be recommended.
Anyway, f-strings are my preference if you are py3.6+ if not, you can just use the fact that print accepts any arguments and will append a space between then.
```suggestion
    print('Found', len(stale_accounts), 'state accounts')
```

Reply:
oooh... unfortunate... I was writing the comment as your update came in... LOL
Well if log.info supports the same thing it might still apply but meh.

Reply:
I'm not a big fan of the `%` syntax either but we're stuck with 3.4 for this sadly

Reply:
oh ew 3.4, that one is end of life already. I am not even sure if string.format was in that or not, that might of been 3.5.